application/cms/filters.py

```diff
@@ -40,6 +40,20 @@ def format_versions(number):
     return "%s&nbsp;versions" % number
 
 
+def index_of_last_initial_zero(list):
```

Comment:
As @TheDoubleK  mentioned on the other PR, this should probably be called `list_` (The convention being, when using variables with the same name as builtins, to append a trailing `_`).

Reply:
reference: https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments